### PR TITLE
Settings shortcut

### LINKED ISSUE #2: No settings shortcut in session menu
I'm not sure whether that's intended or just an accident, but it was a really useful shortcut as it was much faster than opening up slingshot and searching for "System Settings" in your language.
Please bring it back.

Launchpad Details: [#LP1603686](https://bugs.launchpad.net/bugs/1603686) nigdyci - 2016-07-16 19:28:37 +0000

### PR DESCRIPTION
**Pull request by davidmhewitt**
Fixes #2 
screenshot from 2017-03-21 19-29-53

### CODE REVIEW

**Comment by danrabbit** [comment]
Sorry to waste your time on this David, it sounds like UX team is -1 on adding this shortcut. Gonna close this PR since the issue is marked "won't fix"

**Comment by davidmhewitt** [comment]
Don't worry. Was 5 minutes work and it's good to be able to visualise these things before making a decision!

File: src/Indicator.vala [hunk 1]
```diff
@@ -154,6 +164,17 @@ public class Session.Indicator : Wingpanel.Indicator {
                 stderr.printf ("%s\n", e.message);
             }
         });
+
+        if (settings != null && settings_appinfo != null) {
```
Comment (donadigo) [inline src/Indicator.vala]: You should move this up to where the creating of AppInfo is.

File: src/Indicator.vala [hunk 2]
```diff
@@ -31,6 +31,8 @@ public class Session.Indicator : Wingpanel.Indicator {
     private Wingpanel.Widgets.Button log_out;
     private Wingpanel.Widgets.Button suspend;
     private Wingpanel.Widgets.Button shutdown;
+    private Wingpanel.Widgets.Button? settings = null;
```
Comment (donadigo) [inline src/Indicator.vala]: Don't declare those instance variables here, they aren't needed for one time initialization.

**Pull request closed without merging**